### PR TITLE
OTC-484: Fix renewal through USSD to use the bulk CN

### PR DESCRIPTION
**Pull request by hirensoni913**
https://openimis.atlassian.net/browse/OTC-484

### CODE REVIEW

**Review by dragos-dobre — APPROVED** [review]

File: OpenImis.ePayment/Logic/PaymentLogic.cs [hunk 1]
```diff
@@ -570,10 +570,10 @@ public List<BulkControlNumbersForEO> GetControlNumbersForEO(string officerCode,
             return imisPayment.GetControlNumbersForEO(officerCode, productCode);
         }
 
-        public int ControlNumbersToBeRequested(string productCode)
+        public async Task<int> ControlNumbersToBeRequested(string productCode)
         {
             var imisPayment = new ImisPayment(_configuration, _hostingEnvironment);
-            return imisPayment.ControlNumbersToBeRequested(productCode);
+            return await imisPayment.ControlNumbersToBeRequested(productCode);
```
Comment (dragos-dobre) [inline OpenImis.ePayment/Logic/PaymentLogic.cs:576]: This line generates an error on compilation. Please check and fix.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: OpenImis.DB.SqlServer/TblPolicy.cs [hunk 2]
```diff
@@ -31,6 +31,7 @@ public TblPolicy()
         public byte[] RowId { get; set; }
         public bool? IsOffline { get; set; }
         public Guid PolicyUUID { get; set; }
+        public bool SelfRenewed { get; set; }
```
Comment (dragos-dobre) [inline OpenImis.DB.SqlServer/TblPolicy.cs:34]: Was this field added in DB scripts?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: OpenImis.DB.SqlServer/TblPolicy.cs (hunk 2 above)
Comment (hirensoni913) [reply OpenImis.DB.SqlServer/TblPolicy.cs:34]: yes it is. I confirmed in the develop branch. It is merged.

File: OpenImis.ePayment/Logic/PaymentLogic.cs (hunk 1 above)
Comment (hirensoni913) [reply OpenImis.ePayment/Logic/PaymentLogic.cs:576]: How? It compiles well on my machine. In fact the same version is deployed on the production. Tested and confirmed.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**